### PR TITLE
Dont send multiple emails for every donations after payout

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related #476

### CODE REVIEW

File: src/utils/conversationCreator.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const { CONVERSATION_MESSAGE_CONTEXT } = require('../models/conversations.model'
 const { getTransaction } = require('../blockchain/lib/web3Helpers');
 const {
   findSimilarDelegatedConversation,
-  findSimilarPayoutConversation,
+  findPayoutConversationByTxHashAndMilestoneId,
```
Comment (aminlatifi) [inline src/utils/conversationCreator.js:8]: Let's keep its name simple as before, arguments are self descriptive and don't need to be included in function name

File: src/repositories/donationRepository.js [hunk 2]
```diff
@@ -31,4 +31,22 @@ const updateBridgePaymentAuthorizedTxHash = async (
...
+      $exists: false,
+    },
+  });
+  return notPaidOutDonationsCount === 0;
+};
+
+module.exports = {
+  updateBridgePaymentExecutedTxHash,
+  updateBridgePaymentAuthorizedTxHash,
+  isAllDonationsPaidOutForMilestoneAndTxHash,
```
Comment (aminlatifi) [inline src/repositories/donationRepository.js:51]: Please choose simpler name, it's not need to have arguments' name in function name too

**Review by aminlatifi — CHANGES_REQUESTED** [review]

File: src/utils/conversationCreator.js (hunk 1 above)
Comment (mohammadranjbarz) [reply src/utils/conversationCreator.js]: @aminlatifi  done.

File: src/repositories/donationRepository.js (hunk 2 above)
Comment (mohammadranjbarz) [reply src/repositories/donationRepository.js]: @aminlatifi  done

**Pull request merged**